HPXMLtoOpenStudio/tests/test_hvac.rb

```diff
@@ -1170,7 +1182,7 @@ def test_heat_pump_resnet_defrost
 
     # Ductless heat pump test
     args_hash = {}
-    args_hash['hpxml_path'] = File.absolute_path(File.join(@sample_files_path, 'base-hvac-mini-split-heat-pump-ductless.xml'))
+    args_hash['hpxml_path'] = File.absolute_path(File.join(@sample_files_path, 'base-hvac-mini-split-heat-pump-ductless-backup-standard-defrost.xml'))
```

Comment:
This and the next change are the same as above, but for the standard defrost model.